topics/statistics/tutorials/flexynesis_classification/tutorial.md

```diff
@@ -0,0 +1,863 @@
+> > * A PCA plot of the Embeddings colored by predicted subtypes
+> > ![PCA plot - colored by predicted subtypes](../../images/flexynesis_classification/niter5_pca_predicted.jpeg "PCA plot - colored by predicted subtypes")
+> > We can see that the subtypes were predicted by the model with a good approximation.
+> > * A precision-recall curves plot, showing the accuracy of the prediction
+> > ![PR-curve plot - showing prediction accuracy](../../images/flexynesis_classification/niter5_pr_curve.jpeg "PR-curve plot - showing prediction accuracy")
+> > This plot also conforms that most of the subtypes were classified with a good precision. Also the precision has incresed (compare Class4 = 0.23 with 1 iteration to Class4 = 0.41 with 5 iterations)
+> {: .solution}
+>
+{: .question}
+
+## Generate UMAP plot of the embeddings
+
+To generate a UMAP plot, we need to extract the embeddings and predicted table.
+
+> <hands-on-title> Extract embeddings and prediction </hands-on-title>
+>
+> 1. {% tool [Extract dataset](__EXTRACT_DATASET__) %} with the following parameters:
+>    - {% icon param-file %} *"Input List"*: `results` (output of **Flexynesis** {% icon tool %})
+>    - *"How should a dataset be selected?"*: `Select by element identifier`
+>        - *"Element identifier:"*: `job.embeddings_test`
+>
+> 2. {% tool [Extract dataset](__EXTRACT_DATASET__) %} with the following parameters:
+>    - {% icon param-file %} *"Input List"*: `results` (output of **Flexynesis** {% icon tool %})
+>    - *"How should a dataset be selected?"*: `Select by element identifier`
+>        - *"Element identifier:"*: `job.predicted_labels`
+>
+{: .hands_on}
+
+> <hands-on-title> Remove probabilities (duplicates) </hands-on-title>
+>
+> 1. {% tool [Sort](toolshed.g2.bx.psu.edu/repos/bgruening/text_processing/tp_sort_header_tool/9.5+galaxy2) %} with the following parameters:
+>    - *"Sort Query"*: `job.predicted_labels` (output of **Extract dataset** {% icon tool %})
+>    - *"Number of header lines"*: `1`
+>    - In *"Column selections"*:
+>        - {% icon param-repeat %} *"Insert Column selections"*
+>            - *"on column"*: `Column: 1`
+>            - *"Flavor"*: `Alphabetical sort `
+>    - *"Output unique values"*: `Yes`
+>
+{: .hands_on}
+
+And finally the UMAP plot.
+
+> <hands-on-title> UMAP plot </hands-on-title>
+>
+> 1. {% tool [Flexynesis plot](toolshed.g2.bx.psu.edu/repos/bgruening/flexynesis_plot/flexynesis_plot/0.2.20+galaxy3) %} with the following parameters:
+>    - *"I certify that I am not using this tool for commercial purposes."*: `Yes`
+>    - *"Flexynesis plot"*: `Dimensionality reduction`
+>        - {% icon param-file %} *"Predicted labels"*: `table` (output of **Sort** {% icon tool %})
+>        - {% icon param-file %} *"Embeddings"*: `job.embeddings_test` (output of **Extract dataset** {% icon tool %})
+>        - *"Column in the labels file to use for coloring the points in the plot"*: `Column: 5`
```

Comment:
Please specify `Column 5` just like was done for `Column 16` while running Flexynesis